snippets/Responsive-Nuemorphism-Card-Design.md

```diff
@@ -0,0 +1,176 @@
+
+---
+title: Responsive Nuemorphism Card Design
+tags: interactivity,layout
+---
+
+Creates an awesome nuemorphism effect with cool interactivity to the users. 
+
+- Uses `display: flex` property along with `justify-content: center; align-items: center;` to make the snippet responsive.
+- Creates Nuemorphism effect by using nuemorphism colors like `rgba(255, 255, 255, 0.1)` or `rgba(0, 0, 0, 0.6)` and using nuemorphism shadows. 
+- Uses `:hover` pseudo-selector to make the transitions and interactivity.
+
+```html
+<div class="container">
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>01</h2>
+        <h3>Card One</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>02</h2>
+        <h3>Card Two</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>03</h2>
+        <h3>Card Three</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+</div>
+```
+
+```css
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700;800&display=swap");
+* {
```

Comment:
Avoid `*` and `body` selectors as they will affect the whole page and are hard to override or scope.